api/src/main/java/org/pmiops/workbench/api/OfflineRuntimeController.java

```diff
@@ -30,23 +51,46 @@
 public class OfflineRuntimeController implements OfflineRuntimeApiDelegate {
   private static final Logger log = Logger.getLogger(OfflineRuntimeController.class.getName());
 
+  // On the n'th day of inactivity on a PD, a notification is sent.
+  private static final Set<Integer> INACTIVE_DISK_NOTIFY_THRESHOLDS_DAYS = ImmutableSet.of(14);
```

Comment:
Normally a value like this would be in the config, but I have no expectation that we will need the value to vary across environment, or that we'll need to change this out of band at this time. This can be moved to the config later if we decide it makes sense.